docs/Updating Projects.md

```diff
@@ -291,3 +291,28 @@ is invoked by setting `MIX_TARGET` to value other then `host`. Update your
   end
 
 ```
+
+## Updating from v1.0.0-rc.0 to v1.0.0-rc.2
+
+
```

Comment:
FYI - Markdown lint complains about two consecutive blank lines. Don't waste time fixing this, since my editor will make me fix it the next time that I open this file.